SPECS/python-botocore/python-botocore.spec

```diff
@@ -0,0 +1,1860 @@
+Summary:        Low-level, data-driven core of boto 3
+Name:           python-%{pypi_name}
+# NOTICE - Updating this package requires updating python-boto3
+Version:        1.24.33
+Release:        2%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/boto/botocore
+Source0:        https://github.com/boto/botocore/archive/refs/tags/%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+%global pypi_name botocore
+# Replace removed inspect.formatargspec with inspect.signature
+# This is needed for compatibility with Python 3.11 and
+# can be removed once upstream PR is merged.
+Patch2507:      https://github.com/boto/botocore/pull/2507.patch
+BuildArch:      noarch
+
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python3
+BuildRequires:  python3-pip
+BuildRequires:  python3-devel
+BuildRequires:  python3-jsonschema
+BuildRequires:  python-mock
+BuildRequires:  python3-setuptools
+BuildRequires:  python-wheel
+BuildRequires:  python3-jmespath
+
+%if %{with_check}
+BuildRequires:  python3-pytest
+%endif
+
+%description
+A low-level interface to a growing number of Amazon Web Services. The
+botocore package is the foundation for the AWS CLI as well as boto3.
+
+%package -n     python3-%{pypi_name}
+Summary:        Low-level, data-driven core of boto 3
+Provides:       bundled(python3-six) = 1.10.0
+%{?python_provide:%python_provide python3-%{pypi_name}}
+
+
+%description -n python3-%{pypi_name}
+A low-level interface to a growing number of Amazon Web Services. The
+botocore package is the foundation for the AWS CLI as well as boto3.
+
+%prep
+%autosetup -n %{pypi_name}-%{version} -p1
+rm -vr %{pypi_name}.egg-info
+# Remove online tests
+rm -vr tests/integration
+# This test tried to import tests/cmd-runner which failed as the code was
+# unable to import "botocore". I'm not 100% sure why this happened but for now
+# just exclude this one test and run all the other functional tests.
+rm -vr tests/functional/leak
+
+%generate_buildrequires
+# -r use final runtime dependencies as BuildRequires
+%pyproject_buildrequires -r
+
+%build
+%pyproject_wheel
+
+%install
+%pyproject_install
+
+%check
+%pytest3
```

Comment:
I don't think we have this macro in Mariner.

Reply:
This is a typo. Corrected.